src/Components/HomeScreen.tsx

```diff
@@ -1,13 +1,13 @@
 import React from "react"
-import { PlayerDetails } from "./PlayerDetails"
+import { PlayerDetails } from "./HomeScreenComponents/PlayerDetails"
 
 function HomeScreen(){
     return (
         <div>
             <h1>CONNECT-4</h1>
-            <PlayerDetails playerNumber="1"/>
-            <PlayerDetails playerNumber="2"/>
-            <button>PLAY</button>
+            <PlayerDetails playerNumber={1}/>
```

Comment:
Similarly, you could have a constant `const NUM_PLAYERS = 2` in `src/constants.ts`